src/Kuzzle.js

```diff
@@ -29,6 +30,7 @@ function Kuzzle (host, options, cb) {
   if (!(this instanceof Kuzzle)) {
     return new Kuzzle(host, options, cb);
   }
+  KuzzleEventEmitter.call(this);
```

Comment:
nitpicking: if it's not a constructor, then it should not start with a capital K

edit: I'm confused. A few lines below there is a `new KuzzleEventEmitter()`... so what is called here exactly?